src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -532,8 +519,9 @@ def utask_main(uworker_input):
     testcase.set_metadata('minimization_phase', MinimizationPhase.MAIN_FILE)
 
     if time.time() > test_runner.deadline:
-      tasks.add_task('minimize', testcase.key.id(), uworker_input.job_type)
-      return
+      tasks.add_task('minimize', uworker_input.testcase_id,
```

Comment:
nope! i only moved the first chunk of steps and the last step from main. I was planning to move bits out of main each CL but im starting to think it might not have been a perfect idea lmao.

added a TODO!